framework/source/class/qx/ui/tabview/TabView.js

```diff
@@ -563,7 +563,6 @@ qx.Class.define("qx.ui.tabview.TabView",
       {
         value = [button.getUserData("page")];
         pane.setSelection(value);
-        button.focus();
```

Comment:
hmmm that might produce a usability regression ... one might expect that a newly added tab gets the focus ... I agree that it is not ideal to 'force'  this, so maybe it should be either configurable or at least be mentioned in the documentation